sandy-city-utah/sandy-city-types
# feat(types): create package for types

### What Changed?
  - add type files to be consumed by internal projects

### Issues Addressed:
  - P21312-15

### Issue Testing:
  - Manual Developer Testing

### Test Results
1. Set up this project as mentioned in the readme
1. Attempt to run projects that use these types (ad api, ts user sync process)
1. See that the types are found and the projects can run
